sass/_component_toc.scss

```diff
@@ -1,13 +1,17 @@
 .toc {
   a {
-    position: relative;
     display: inline-block;
     line-height: 1.25;
     padding: ($spacer / 4) 0;
   }
 
   li {
     margin: $size-1 0;
+
+    &[class^="toctree-"] {
+      // ensure the toctree-expand button can be positioned
```

Comment:
Not the nicest selector - but otherwise we have to make every `li` position relative.

Reply:
That selector is fine, easy enough to understand. You're working with the HTML that sphinx outputs.

Reply:
Thanks